public/content/sv/welcome/install-d-locally.md

```diff
@@ -0,0 +1,43 @@
+# Installera D lokalt
+
+På D språkets webbsida [dlang.org](https://dlang.org) kan den senaste kompilerar
+versionen av referens kompilatorn **DMD** (Digital Mars D)
+[laddas ned](http://dlang.org/download.html) och installeras:
+
+### Windows
+
+* [Installerare](http://downloads.dlang.org/releases/2.x/{{latest-release}}/dmd-{{latest-release}}.exe)
+* eller: [Arkiv](http://downloads.dlang.org/releases/2.x/{{latest-release}}/dmd.{{latest-release}}.windows.7z)
+* genom [chocolatey](https://chocolatey.org/packages/dmd): `choco install dmd`
+
+### Mac OS X
+
+* `.dmg` [fil](http://downloads.dlang.org/releases/2.x/{{latest-release}}/dmd.{{latest-release}}.dmg)
+* eller: [Arkiv](http://downloads.dlang.org/releases/2.x/{{latest-release}}/dmd.{{latest-release}}.osx.tar.xz)
+* genom [Homebrew](http://brew.sh): `brew install dmd`
```

Comment:
Remove `eller:` and `genom`, it doesn't add anything. It's a correct translation but I don't see the reason to have it in the English version either.